src/guides/v2.3/performance-best-practices/deployment-flow.md

```diff
@@ -9,24 +9,14 @@ functional_areas:
 
 The Magento production deployment flow helps a store reach maximum performance.
 
-## Deploy static content
-
-Deploying static content causes Magento 2 to perform the following actions:
-
-*  Analyze all static resources
-*  Perform merge, minimization, and bundling of content
-*  Read and process theme data
-*  Analyze theme fallback
-*  Store all processed and materialized content to specific folder for further usage
+## Install dependencies
 
-If your static content is not deployed, Magento performs all listed operation on the fly, leading to a significant increase in response time.
-
-You can use a variety of options to customize deployment operations based on store size and fulfillment needs. The most common is the compact deploy strategy. See [Static files deployment strategies]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html)
+The `composer.json` and `composer.lock` files tell Composer what Magento dependencies to resolve and which versions to use (respectively).
```

Comment:
Anthropomorphic. Might be better to say the files manage the dependencies, as opposed to creating a visual of the files shaking a finger at Composer and Magento.

Reply:
Perhaps something like:

```suggestion
The `composer.json` and `composer.lock` files manage Magento dependencies and install the appropriate version for each package.
```